chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -660,13 +660,55 @@ include::../../validity/protos/vkAcquireNextImageKHR.txt[]
 When successful, fname:vkAcquireNextImageKHR acquires a presentable image
 from pname:swapchain that an application can: use, and sets
 pname:pImageIndex to the index of that image within the swapchain.
-The presentation engine may: not have finished reading from the image at the
-time it is acquired, so the application must: use pname:semaphore and/or
-pname:fence to ensure that the image layout and contents are not modified
-until the presentation engine reads have completed.
 The order in which images are acquired is implementation-dependent, and may:
 be different than the order the images were presented.
 
+fname:vkAcquireNextImageKHR is similar to a <<devsandqueues-submission,queue
+submission command>>, but it does not require a slink:VkQueue as a
+parameter.
+It does have a single _batch_ of work: waiting for presentation engine to
+finish reading the image.
+And it does have a zero or one slink:VkSemaphore, and zero or one
+slink:VkFence, and defines semaphore and fence wait operation on them,
+respectively.
+
+[NOTE]
+.Note
+====
+Exclusive ownership of presentable images corresponding to a swapchain
+created with ename:VK_SHARING_MODE_EXCLUSIVE as defined in
+<<resources-sharing,Resource Sharing>> is not altered by a call to
+fname:vkAcquireNextImageKHR.
+That means upon the first acquisition from such a swapchain presentable
+images are not owned by any queue family, while at subsequent acquisitions
+the presentable images remain owned by the queue family the image was
+previously presented on.
+====
+
+Presentation engine may: still read from the image after it is acquired.
+pname:semaphore, and pname:fence will become signaled when the presentation
+engine is done reading from the image, and the application can: safely
+access it.
+An application must: therefore wait until either the pname:semaphore or the
+pname:fence is signaled before accessing the image's data.
+flink:vkDeviceWaitIdle called with identical pname:device must: cause the
+_batch_ to finish execution, and pname:fence to become signaled.
```

Comment:
Not 100 % sure if this is actually true.
Also don't know what the interaction with `vkQueueWaitIdle` is, but it should be included.